### PR TITLE
Always have the header customization

### PR DESCRIPTION
**Pull request by bsipocz**
we import the display anyway so these should work out of the box

### CODE REVIEW

**Comment by bsipocz** [comment]
(this is one more thing that would be nice to have before we announce the new tags)

File: {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py [hunk 1]
```diff
@@ -27,31 +28,20 @@
...
-#     del PYTEST_HEADER_MODULES['h5py']
-# except (NameError, KeyError):  # NameError is needed to support Astropy < 1.0
-#     pass
+# Customize the following lines to add/remove entries from
+# the list of packages for which version numbers are displayed when running
+# the tests. Making it pass for KeyError is essential in some cases when
+# the package uses other astropy affiliated packages.
+try:
+    PYTEST_HEADER_MODULES['Astropy'] = 'astropy'
+    del PYTEST_HEADER_MODULES['h5py']
```
Comment (pllim) [inline {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py]: Should we comment out the `del` part by default? If a package uses `h5py`, they will be very confused why `h5py` is not showing?

Comment (bsipocz) [reply {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py]: it's an example, they should be able to remove the line, and add other lines based on their dependencies. Maybe we should preload the deletion of all the packages being there by default (at they are a dependency of astropy?)?

Comment (pllim) [reply {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py]: Naw, it's no big deal. It was commented before, so I am just wondering.

Comment (bsipocz) [reply {{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py]: yes, I removed the comment as many projects didn't bother uncomment them. So making the default to have the display on is to make my life easier if I ever need to look at their test logs 😅 

If they decide they don't like these headers that's cool, too, but that needs to be a decision :)

**Pull request merged**